www/src/config.ts

```diff
@@ -26,6 +26,7 @@ export type Frontmatter = {
 
 export const KNOWN_LANGUAGES = {
   en: "🇺🇸 English",
+  it: "🇮🇹 Italian"
```

Comment:
Should be "Italiano" I think? We are using the name of the language, in that language.

Reply:
Oh ok missunderstud!